sw/device/tests/consecutive_irqs_test.c

```diff
@@ -17,7 +17,6 @@
 #define PLIC_TARGET kTopEarlgreyPlicTargetIbex0
 
 #define PLIC_PRIORITY_MIN 0u
```

Comment:
Do you think it is reasonable to add `kDifPlicMinPriority` to the plic DIF too, or do you think the fact the `kDifPlicMaxPriority` is unsigned (and so are the `priority` arguments) is enough of a clue that 0 is the minimum?

Reply:
> is enough of a clue that 0 is the minimum?

Yes, but for aesthetics alone, I will just add it (there's no harm in it).

Reply:
Dones.